src/errors.ts

```diff
@@ -287,8 +287,8 @@ export const errorMessages: {
   [Errors.IllegalReturn]: 'Illegal return statement',
   [Errors.InvalidForLHSBinding]: 'The for-header left hand side binding declaration is not destructible',
   [Errors.InvalidNewTarget]: 'new.target only allowed within functions',
-  [Errors.InvalidEscapedKeyword]: 'Keywords must be written literally, without embedded escapes',
-  [Errors.MissingPrivateName]: "''#' not followed by identifier",
+  [Errors.InvalidEscapedKeyword]: 'Kasdfafdsadfsy, without embedded escapes',
```

Comment:
the message looks confusing. is it expected?

Reply:
`[Errors.InvalidEscapedKeyword` will be gone and the `Errors.MissingPrivateName` seems to have an typo - an extra `'`. 

What's confusing? Everything can be changed :)

Reply:
what's the meaning of `Kasdfafdsadfsy`

Reply:
Just a typo :) It's all fixed